packages/server/src/organization/organization.service.ts

```diff
@@ -16,6 +16,18 @@ export class OrganizationService {
     return organizationUser.role;
   }
 
+  public async getOrganizationIdForUser(userId: number): Promise<number> {
```

Comment:
useProfile should return this information on frontend?

Reply:
I don't know what you mean. This function is called in `profile.controller` so that the `organizationId` can be returned with the profile. Are you saying useProfile should not return this?